roles/pulp_common/vars/CentOS-7.yml

```diff
@@ -5,7 +5,7 @@ pulp_preq_packages:
   - python-setuptools
   - libselinux-python  # For ansible itself
   - policycoreutils-python # For ansible itself
-  - postgresql-devel
+  - rh-postgresql10-postgresql-devel
```

Comment:
pulp_common now needs to enable the SCL. Otherwise, running pulp_common on one node, and pulp_database on another, will not work.

Are you sure this change is necessary? Are the symlinks you now created in the pulp_database role interfering with the dependencies of postgresql-devel ?

Note that a task on my to-do list has been to run separate containers for the "dynamic" molecule CI tests. It would catch this issue.

Another task has been to create a pulp_repos role, and move the repo logic into that. My idea is that roles (pulp_common, pulp_redis, and pulp_database) can statically depend on pulp_repos, and they can set / pass boolean vars if they need a repo.

Reply:
`/opt/rh/rh-postgresql10/root/usr/bin/pg_config` is from `rh-postgresql10-postgresql-devel`, the other approach would be keeping `postgresql-devel`.
`pg_config` is needed when pip install pulp_plugins from source

Reply:
So I just checked, postgresql-devel depends on postgresql, which provides /usr/bin/pg_config. So your symlinks would conflict.